content/docs/communities/administration/_index.md

```diff
@@ -0,0 +1,166 @@
+You can also downgrade and sidegrade in this way.
+
+**Limitations:**  
+While the [Matrix specification](https://spec.matrix.org/v1.15/client-server-api/#server-behaviour-19) is not overly strict about it, your homeserver will execute the room upgrade on a best effort basis, trying to create the new room with settings matching the old room.
+The decentralised nature of Matrix can lead to circumstances that cannot be covered automatically by your homeserver.
+We suggest considering the following recommendations to mitigate this.
+
+#### Recommendations
+
+This is a non-exhaustive list of recommendations to consider when planning a room upgrade.
+Which of these cases apply to your room depends on your specific setup, for example size of your community.
+
+**Choose the "right" account on the "right" homeserver** to execute your upgrade.  
+The choice of the account starting the upgrade implies the homeserver that will perform the upgrade.
+Here are some important factors to guide your choice:
+
+1. Upgrading a room means that the room members need to receive the info about the upgrade and then join the new room.
+   The homeserver performing the upgrade by necessity becomes the center of attention here for a while, acting as a gateway to the new room in a sense.
+   It thus makes sense to choose a homeserver that is fast and well connected so it can handle all the joins following the room upgrade itself well.
+   If your community is centered around a certain homeserver, it probably makes sense use that one.
+2. You as well as users on other homeservers may have set up [aliases](https://spec.matrix.org/latest/client-server-api/#room-aliases) - shareable addresses to find the room that are hosted by a specific homeserver.
+   By definition, a homeserver can thus only update aliases itself hosts.
+   Thus only aliases on the same homeserver will be transferred automatically.
+   However you can (and should) always update the aliases on other homeservers after the upgrade.
+3. If your community is using a space to organise its rooms, the user upgrading a room needs at least enough access to the parent space to update the reference from the old to the upgraded room.
+4. Room version 12 introduces some changes to room ownership semantics, such as [irrevokable full control](https://github.com/matrix-org/matrix-spec-proposals/pull/4289) of the room.
+   Using a separate service account to execute the upgrade means that this account will be the creator of the new room with the special access rights.
+   It being a special account means you can manage access to it among your admins independently of the rules Matrix imposes on you.
+   You could also consider setting up multiple such accounts across multiple homeservers as [`additional_creators`](https://github.com/matrix-org/matrix-spec-proposals/pull/4289), in case one of them becomes unavailable.
```

Comment:
```suggestion
4. Room version 12 introduces some changes to room ownership semantics, such as [irrevocable full control](https://github.com/matrix-org/matrix-spec-proposals/pull/4289) of the room.
   For communities, you should use a long-lived, trusted account, such as a moderation bot account, to upgrade your rooms. Using a separate service account to execute the upgrade means that this account will be the creator of the new room with the new special access rights.
   As a special account, you can manage access to it among your admins outside of Matrix.
   You could also consider setting up backup accounts on homeservers as [`additional_creators`](https://github.com/matrix-org/matrix-spec-proposals/pull/4289), in case one of them becomes unavailable.
```